### PR TITLE
add ui.date and ui.slider modules to gmf

### LINKED ISSUE #1274: Refactor all gmf examples to include new angular-ui dependencies
We included new dependancies to gmf in order to handle time selection for WMS layers #1241 

This will add two dependencies for the gmf angular module:

- 'ui.slider' for the slider
- 'ui.date' for the datepicker

The module declaration will be:
`gmf.module = angular.module('gmf', [ngeo.module.name, 'gettext',
    'ngAnimate', 'ngTouch', 'ngSanitize', 'ui.date', 'ui.slider']);`

Therefore all examples using the gmf angular module should be able to retrieve those modules.

### PR DESCRIPTION
**Pull request by olivierSemet**
@sbrunner @fredj 

I don't now how to add the new folder contribs/gmf/third-party to the build process @sbrunner 

close #1274

### CODE REVIEW

**Comment by olivierSemet** [comment]
@sbrunner I don't know why all test fail...

**Comment by fredj** [comment]
> I don't now how to add the new folder contribs/gmf/third-party to the build process @sbrunner

```
build/examples-hosted/lib/jquery-ui.min.js: contribs/gmf/third-party/jquery-ui/jquery-ui.min.js
 	mkdir -p $(dir $@)
 	cp $< $@
```

`mkdir -p` creates the directory; is it's still an issue ?

**Comment by olivierSemet** [comment]
@fredj  no, As I understand Makefile, this rule will create the folder:

> build/examples-hosted/lib/

- mkdir -p $(dir $@) means "take the folder part of the target and create it.
cp $< $@ means "take the first dependency of the target and copy it to the freshly created folder

But we never create a folder build/examples-hosted/contribs/gmf/third-party. Instead the jquery-ui dependency has been placed like other node_modules dependencies in build/examples-hosted/lib/

What I mean is we changed the folder structure between dev and build. Since I appended two sed commands telling that the path:
> /contribs/gmf/third-party/jquery-ui/jquery-ui.min\.js 
should be changed by 
> lib/jquery-ui.min.js

during the build process, everything works. But maybe @sbrunner  will not like it

**Comment by olivierSemet** [comment]
@fredj  can we merge?

**Comment by fredj** [comment]
thanks @olivierSemet

File: Makefile [hunk 1]
```diff
@@ -413,12 +431,15 @@ node_modules/angular/angular.min.js: .build/node_modules.timestamp
 		-e 's|\.\./node_modules/bootstrap/dist/css/bootstrap.css|lib/bootstrap.min.css|' \
 		-e 's|\.\./node_modules/font-awesome/css/font-awesome.css|lib/font-awesome.min.css|' \
 		-e 's|\.\./node_modules/jquery/dist/jquery.js|lib/jquery.min.js|' \
+		-e 's|\.\./contribs/gmf/third-party/jquery-ui/jquery-ui.min.js|lib/jquery-ui.min.js|' \
```
Comment (sbrunner) [inline Makefile]: We add gmf thing in non gmf example :-/

**Comment by sbrunner** [comment]
Shouldn't we also add `contribs/gmf/third-party/jquery-ui/jquery-ui.min.js` in the gmf examples?

**Comment by olivierSemet** [comment]
@sbrunner  yes I changed the Makefile in #1320

**Comment by sbrunner** [comment]
Yes :-), but the example should also be modified :-)

**Pull request merged**